### DISCUSSION TITLE
An update on my commitment to xmoj-script development

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#467] [General]
![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/62ed2f20-c651-4293-a98c-9ca300f020a8)

#463 will be the last pull request I make to this project until this July.
Until then, I will not make any code contributions to this project.
Thank you for your understanding.
@PythonSmall-Q @langningchen @zhouyiqing0304

**Comment by PythonSmall-Q** [comment]
嗯，我也在平衡我的个人生活，学习 ~和感情生活~ ，我也可能要摆了ovo

**Comment by langningchen** [comment]
一切祝好

**Comment by boomzero** [comment]
你不一直在摆吗（在 2024年2月17日，20:22，Shan Wenxiao ***@***.***> 写道：﻿
嗯，我也在平衡我的个人生活，学习 和感情生活 ，我也可能要摆了ovo

**Comment by langningchen** [comment]
我估计以后也会减少上线GitHub的频率，所有回复不一定能及时，同时我也要取消watch这个仓库

**Comment by boomzero** [comment]
Well, I hope xmoj/script still receives updates and maintenance在 2024年2月17日，22:10，Langning Chen ***@***.***> 写道：﻿
我估计以后也会减少上线GitHub的频率，所有回复不一定能及时，同时我也要取消watch这个仓库
